### ISSUE TITLE
Unable to start Docker facilmap 4.1.2 and mariadb 11.4

### ISSUE THREAD

**Issue by grisumedia** [FacilMap/facilmap#290]
Hi there, I'm running facilmap in Docker 27.0.3. I haven't visited my facilmap instance a few weeks, but now it is constantly crashing.
```
facilmap-1  | (node:46) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
facilmap-1  | (Use `node --trace-deprecation ...` to show where the warning was created)
facilmap-1  | file:///opt/facilmap/server/dist/facilmap-server.mjs:4150
facilmap-1  |     if (markersAttributes.symbol.allowNull) {
facilmap-1  |                                  ^
facilmap-1  | 
facilmap-1  | TypeError: Cannot read properties of undefined (reading 'allowNull')
facilmap-1  |     at DatabaseMigrations._changeColMigrations (file:///opt/facilmap/server/dist/facilmap-server.mjs:4150:34)
facilmap-1  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
facilmap-1  |     at async DatabaseMigrations._runMigrations (file:///opt/facilmap/server/dist/facilmap-server.mjs:4068:5)
facilmap-1  |     at async Database.connect (file:///opt/facilmap/server/dist/facilmap-server.mjs:4578:5)
facilmap-1  |     at async startServer (file:///opt/facilmap/server/dist/facilmap-server.mjs:6185:3)
facilmap-1  |     at async file:///opt/facilmap/server/bin/facilmap-server.js:5:1
facilmap-1  | 
facilmap-1  | Node.js v21.7.3
mariadb-1   | 2024-07-10 16:20:06 6 [Warning] Aborted connection 6 to db: 'facilmap' user: 'facilmap' host: '172.18.0.3' (Got an error reading communication packets)
```
The server runs fine if the mariadb database is deleted. So a fresh start is possible and facilmap is working. 
I'm also able to dump my data from mariadb, but still, facilmap won't start when my data is in the database.
Any help is appreciated.

**Comment by cdauth** [comment]
Could it be that you ran the `latest` docker image in the meantime and are now trying to run `4.1.2`? I'm asking because the `latest` image currently contains an alpha version of FacilMap v5. If you started it, it already migrated the database layout to the v5 format, which would fail with v4. (Sorry, the versioning is a mess right now. But the alpha version of v5 is stable and running on facilmap.org, there are just some features missing before it can be released.)

Could you check the columns of the `Markers` table? If if contains a column `icon`, you are already on the v5 layout. In that case I would recommend you to use the `latest` docker image until v5 is released, then you can switch back to a fixed version number.

**Comment by grisumedia** [comment]
I have tried different versions of facilmap. My oldest backup is created with `latest` facilmap on 2024-05-28, which may already be 5.0.0 as the `Markers` table has an `icon` column. So no switch back to 4.1 for me, that's okay.

I have assumed that `latest` is 4.1.2, so my problem refers to 5.0.0-alpha instead.

But using `latest` facilmap creates that problem. To make things clearer: I have erased my database, moved my backup from 2024-05-28 in place, started `latest` facilmap and the error message comes up.

**Comment by cdauth** [comment]
So I investigated this a little bit, and it turns out that there was an error in the GitHub Action and it was pushing to `latest` not just on updates to the main branch but also on releases. So when I released 4.1.2, `latest` got downgraded from v5 to v4, and now you were trying to start v4 with a v5 database.

I have fixed the configuration and `latest` contains v5 again. If you pull the latest image, it should work again.
